RELEASE_NOTES.md

```diff
@@ -4,6 +4,7 @@ _This file holds "in progress" release notes for the current release under devel
 
 ## Highlighted enhancements for this release:
 
+- Added `returns` property to `chocolatey_package` resource
```

Comment:
These highlights tell a little more of a story than the changelog, how about:

You can now specify the acceptable return codes from the `chocolatey_package` resource using the `returns` property.